R/utilities-parameter.R

```diff
@@ -46,3 +63,66 @@ toParameters <- function(netParams) {
   sapply(netParams, function(p)
     Parameter$new(p))
 }
+
+#' Set values of parameters
+#'
+#' @param parameters A single object of type 'Parameter' or a list of such objects
+#' The objects are created by calling 'getParameter' or 'getAllParametersMatching'
+#'
+#' @param values A numeric value that should be assigned to the parameter or a vector
+#' of numeric values, if the value of more than one parameter should be changed. Must have the same
+#' length as 'parameters'
+#'
+#' @examples
+#'
+#' #' simPath <- system.file("extdata", "simple.pkml", package = "ospsuite")
+#' sim <- loadSimulation(simPath)
+#' param <- getParameter(c("Organism", "Liver", "Volume"), sim)
+#' setParametersValues(param, 1)
+#' params <- getAllParametersMatching(c("Organism", "Liver", "*", "Volume"), sim)
+#' setParametersValues(params, c(1:6))
+#'
+#' @export
+setParametersValues <- function(parameters, values){
+  singleParameter <- FALSE
+
+  #Test if 'parameter' is an instance of the Parameter-class.
+  if (inherits(parameters, "Parameter") && inherits(parameters, "ObjectBase")){

```

Comment:
Yes, my initial thought was to double-check as "Parameter" might be also some class other than ours (I could image other packages use such classes), but I think it's OK to just check for Parameter or Simulation...

Reply:
That's ok for me if people are using objects that are not part of the package and the app crashes. 

If we try to check for that, 95% of our code will be to prevent those types of mistakes which is just a waste of time on our part. Let's assume people are using the lib in an expected way.

Reply:
Only check for the higher class now.